apps/hexagon_dma/pipeline_err.cpp

```diff
@@ -2,23 +2,23 @@
 
 using namespace Halide;
 
-#define DIM 3
+#define DIM 4
```

Comment:
Don't think it's worth the indirection when reading the code (just put 4 in the Input/Output buffer definitions).

Reply:
Corrected